### PR TITLE
feat: automate adding stale.yml to all repos

### PR DESCRIPTION
**Pull request by galargh**
This PR ensures that whenever we see a new repo, we add `stale.yml` there.

We cannot automate adding `need/author-input` label like that because GitHub Management doesn't support labels **yet**.

### CODE REVIEW

File: scripts/src/actions/add-need-author-input-label-to-all-repos.ts [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import {addLabelToAllRepos} from './shared/add-label-to-all-repos'
+
+addLabelToAllRepos(
+  'need/author-input',
+  'ededed',
+  'Needs input from the original author'
+)
```
Comment (SgtPooki) [inline scripts/src/actions/add-need-author-input-label-to-all-repos.ts:7]: fixes https://github.com/ipfs/github-mgmt/issues/74 ?

File: scripts/src/actions/fix-yaml-config.ts [hunk 2]
```diff
@@ -1,4 +1,6 @@
 import 'reflect-metadata'
+import {addFileToAllRepos} from './shared/add-file-to-all-repos'
 import {format} from './shared/format'
 
+addFileToAllRepos('.github/workflows/stale.yml', '.github/workflows/stale.yml')
```
Comment (SgtPooki) [inline scripts/src/actions/fix-yaml-config.ts:5]: should `addFileToAllRepos` default to/from to the same path so we don't have this clunky interface of having to pass the same path twice?

**Review by SgtPooki — APPROVED** [review]

File: scripts/src/actions/fix-yaml-config.ts (hunk 2 above)
Comment (galargh) [reply scripts/src/actions/fix-yaml-config.ts:5]: Good idea, let me add this.

Comment (galargh) [reply scripts/src/actions/fix-yaml-config.ts:5]: Done in https://github.com/protocol/github-mgmt-template/commit/47fefec4f514a564c9d8002b41bb7d6b6df51f7e

We'll be able to simplify this file after we upgrade GitHub Management here.

**Pull request merged**